### ISSUE TITLE
Add weight units to vehicles

### ISSUE THREAD

**Issue by arbron** [foundryvtt/dnd5e#1237]
Vehicle cargo capacity is measured in tons but that isn't indicated anywhere on the sheet making it confusing when adding items and having the weights not seem to add up. An indicator of the units on the encumbrance bar and next to the `"Cargo Capacity"` field would help resolve this confusion.

**Comment by erikhuck** [comment]
@arbron agreed. The units should be clearly labeled and we should be able to customize the units of measurement. Tons as the default unit for dnd5e doesn't make much sense since almost every mount carrying capacity is some number of pounds.

**Comment by Ackerfe** [comment]
I know this isn't a fix to this request. But from what I can see the weight is defined in short tons (2000 pounds = 1 ton) and while it's true that the capacity for land vehicles is calculated by multiplying the carrying capacity of the pulling animal by 5 and you can't put non-integers in the capacity field on the sheet, what you CAN do is set up the capacity using effects on the sheet, so just go in the effects tab of the sheet and create a new on targeting system.attributes.capacity.cargo and set that to override and give it whatever value you need, just remember to take the calculated pulling capacity of your mule/horse/elephant/etc.. and divide it by 2000 so that you get proper weight calculation in the sheet
